src/services/AsyncStorage.ts

```diff
@@ -7,6 +7,16 @@ const SELECTED_PROFESSIONS_KEY = 'selectedProfessions'
 const CUSTOM_DISCIPLINES_KEY = 'customDisciplines'
 const FAVORITES_KEY = 'favorites'
 const PROGRESS_KEY = 'progress'
+const TRACKING_KEY = 'tracking'
```

Comment:
I am a little bit confused about the naming, why is tracking the same as sentry error reporting?